Packs/GoogleChatViaWebhook/ReleaseNotes/1_0_1.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "Deleted (SpaceDisplayName, SpaceType, CreatedTime, ThreadReply, SenderDisplayName, SenderName, SenderType) to align with Google Chat Webhook API outputs. These outputs were also removed from Context Data outputs in the integration settings."
```

Comment:
please change it to:
Deleted SpaceDisplayName, SpaceType, CreatedTime, ThreadReply, SenderDisplayName, SenderName, SenderType from the outputs to align with the Google Chat Webhook API response change which does not support it.

Reply:
Done